feat(EOL): Added EOL details (#17068)
newrelic/docs-website: src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide.mdx

The synthetics job manager brings monitors on the newest runtime to your private locations, enabling your monitors to more closely match a real user's experience.

> Ready to download the job manager? You have a few options for getting started. Check out our [install doc](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager) for installation procedures, or read through our [configuration doc](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration) for working with new or existing [private locations](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations). 

Reviewer: ```suggestion
If you're ready to install the job manager, you can check out our [install doc](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager) for installation procedures, or read through our [configuration doc](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration) for working with new or existing [private locations](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations). 
```

* Reduces the risk of internal engine errors by no longer creating unique docker networks for non-ping monitor jobs.
* Gives better support at deployment for Kubernetes labels, annotations, node selectors, tolerations, and affinity across all components

> ## Recommended Migration Steps [#monitorMigration]

Reviewer: ```suggestion
## Recommended migration steps [#monitorMigration]
```

> Installing job manager with the same private location key as your existing CPM will simplify the migration process. CPMs will execute monitors that use the legacy runtime, job managers will execute monitors that use the newest runtime, and ping monitors will be load balanced and executed by both the CPM and job managers. 

Reviewer: ```suggestion
When you install the job manager, we recommend that you use the same private location key as your existing CPM. This simplifies the migration process. Keep in mind that:

* CPMs execute monitors that use the legacy runtime
* Job managers execute monitors that use the newest runtime
* Ping monitors are load balanced, so both CPM and job managers can execute them
```

Reviewer: Re: the bulleted lists, why do customers need to know this? Is it so they can more easily keep track of what's being used in their environment?

Author: They need to know what jobs will run from each as they complete the migration from CPM to job manager. This would also help them scale job manager and CPM during the migration if they decide to do so.
